moped-editor/src/views/projects/newProjectView/NewProjectMap.js

```diff
@@ -269,15 +270,14 @@ const NewProjectMap = ({
   const mapGeocoderContainerRef = useRef();
   const mapEditToolsContainerRef = useRef();
   const mapBasemapContainerRef = useRef();
-
   /**
    * Generate a viewport configuration object
    */
   const [viewport, setViewport] = useFeatureCollectionToFitBounds(
     mapRef,
     // If this is a new feature, use the project feature collection to retrieve the area
     newFeature ? projectFeatureCollection : featureCollection,
-    false
+    isSignalComponent
```

Comment:
this causes the map to auto-zoom to selecting a signal during component editing. when creating a new component, the map will not zoom to the selected signal until the component is saved. 

all of this was quite tricky because we're forking the map editing behavior specifically for signal components.